src/main/java/com/textmessenger/utils/PasswordEncoder.java

```diff
@@ -0,0 +1,18 @@
+package com.textmessenger.utils;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+public class PasswordEncoder {
+  public static void main(String[] args) {
+
+    String [] password = {"sarah","johny","samurai","alex007","amilyUta"};
+    String [] hashedPassword = new String[password.length];
+    BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+    for (int i = 0; i < password.length; i++) {
+      hashedPassword[i] = passwordEncoder.encode(password[i]);
+    }
+    for (String s : hashedPassword) {
+      System.out.println(s);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=229&issues=AWYBSD6ovOWp7Yd69RWL&open=AWYBSD6ovOWp7Yd69RWL)

[ISSUE_KEY]: AWYBSD6ovOWp7Yd69RWL